app/controllers/user_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UserController < ApplicationController
```

Comment:
You dont need a users controller, Devise takes care of all of these actions for you